From 6340a1e18aaaa236bb09680fdae5c5c58ec2a75b Mon Sep 17 00:00:00 2001
From: Gerd Hoffmann <kraxel@redhat.com>
Date: Mon, 14 Oct 2013 13:35:10 +0200
Subject: [PATCH 03/18] qemu-socket: drop pointless allocation

RH-Author: Gerd Hoffmann <kraxel@redhat.com>
Message-id: <1381757723-23134-3-git-send-email-kraxel@redhat.com>
Patchwork-id: 54849
O-Subject: [RHEL-7 qemu-kvm PATCH 02/15] qemu-socket: drop pointless allocation
Bugzilla: 922010
RH-Acked-by: Hans de Goede <hdegoede@redhat.com>
RH-Acked-by: Laszlo Ersek <lersek@redhat.com>
RH-Acked-by: Paolo Bonzini <pbonzini@redhat.com>

Signed-off-by: Gerd Hoffmann <kraxel@redhat.com>
Reviewed-by: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Michael Tokarev <mjt@tls.msk.ru>
(cherry picked from commit c1204afc7c048cf58b0a8f1c11886f36ca73ef28)
---
 util/qemu-sockets.c | 1 -
 1 file changed, 1 deletion(-)

Signed-off-by: Miroslav Rezanina <mrezanin@redhat.com>
---
 util/qemu-sockets.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/util/qemu-sockets.c b/util/qemu-sockets.c
index 86fb09c..35023a8 100644
--- a/util/qemu-sockets.c
+++ b/util/qemu-sockets.c
@@ -871,7 +871,6 @@ SocketAddress *socket_parse(const char *str, Error **errp)
         }
     } else {
         addr->kind = SOCKET_ADDRESS_KIND_INET;
-        addr->inet = g_new(InetSocketAddress, 1);
         addr->inet = inet_parse(str, errp);
         if (addr->inet == NULL) {
             goto fail;
-- 
1.7.1

